client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/test/site-content.tsx

```diff
@@ -21,6 +21,11 @@ jest.mock(
 	() => 'span'
 );
 
+jest.mock(
```

Comment:
I had to introduce this mock because with my changes we now retrieve and interpret the result of a test connection request in hook consumer land. Previously the test on line 76 would not render `SitesTableRow` correctly, showing lots of warnings, because it [assumed an error](https://github.com/Automattic/wp-calypso/blob/trunk/client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-table-row/index.tsx#L52) (the hooks to fetch sites never run in this test) but they don't anymore because we retrieve the `connected` state separately now and default to `true` within this component. The tests could be further improved but it's touching areas which I don't think this PR should take care of.